blockchain/state_manager/src/lib.rs

```diff
@@ -1400,8 +1397,10 @@ where
         self: &Arc<Self>,
         height: ChainEpoch,
         state_tree: &StateTree<DB>,
-    ) -> Result<TokenAmount, Box<dyn StdError>> {
-        self.genesis_info.get_supply(height, state_tree)
+    ) -> Result<TokenAmount, anyhow::Error> {
+        self.genesis_info
+            .get_supply(height, state_tree)
+            .map_err(|e| anyhow::anyhow!("{}", e))
```

Comment:
`get_supply` can be updated to use `anyhow::Error`.

Reply:
Was already the case actually.